static/js/topic_page/ranking_tile.tsx

```diff
@@ -30,20 +30,24 @@ import { RankingMetadata } from "./tile";
 const RANKING_COUNT = 5;
 
 interface RankingData {
-  [key: string]: Point[]; // Key is main statVarDcid
+  points: Point[];
+  unit: string;
+  scaling: number;
 }
 
 interface RankingTilePropType {
   id: string;
   placeDcid: string;
   enclosedPlaceType: string;
   title: string;
-  statVarMetadata: StatVarMetadata;
+  statVarMetadata: StatVarMetadata[];
   rankingMetadata: RankingMetadata;
 }
 
 export function RankingTile(props: RankingTilePropType): JSX.Element {
-  const [rankingData, setRankingData] = useState<RankingData | undefined>(null);
+  const [rankingData, setRankingData] = useState<
+    Record<string, RankingData> | undefined
```

Comment:
nit: i might rename the current RankingData to RankingPoint, and wrap the Record<string, RankingPoint> to RankingData. it'll be easier to document the keys of RankingData in that way, update types, etc. we could also pass RankingPoint to RankingUnit component directly.

Reply:
I agree with this approach but I'm not sure RankingPoint is the best name for it because it holds the values for multiple points in the ranking. Maybe RankingGroup?

Reply:
done